packages/core/src/sagas/sendEventToPostActivitySaga.js

```diff
@@ -0,0 +1,27 @@
+import {
+  put,
+  takeEvery
+} from 'redux-saga/effects';
+
+import whileConnected from './effects/whileConnected';
+
+import { SEND_EVENT } from '../actions/sendEvent';
+import postActivity from '../actions/postActivity';
+
+export default function* () {
+  yield whileConnected(function* () {
+    yield takeEvery(
+      ({ payload, type }) => (
```

Comment:
inlining this takes away from the readability for me. can it be defined outside the function and given a good name?